ocs_ci/utility/aws.py

```
@@ -184,6 +184,122 @@ def sts_client(self):
+
+        Args:
+            username (str): IAM username to delete
+
+        """
+        for policy_name in self.iam_client.list_user_policies(UserName=username).get(
+            "PolicyNames", []
+        ):
+            self.iam_client.delete_user_policy(
+                UserName=username, PolicyName=policy_name
+            )
+            logger.info(f"Deleted inline policy {policy_name} from user {username}")
```

Ruff (0.15.14)

[warning] Logging statement uses f-string

(G004)